activities/service/v1/notifications.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.v1";
+option go_package = "activities.v1";
+
+import "google/protobuf/empty.proto";
+import "activities/domain/v1/domain.proto";
+
+service notificationsService {
+  rpc updateNotificationChannelsConfiguration (UpdateNotificationChannelConfiguration) returns (google.protobuf.Empty);
+  rpc getNotificationChannelsConfiguration (google.protobuf.Empty) returns (NotificationChannelsConfigurationResponse);
+}
+
+message UpdateNotificationChannelConfiguration{
```

Comment:
I guess an id and a label should be added here. So we can edit them

Reply:
added